electron/src/js/util.ts

```diff
@@ -37,4 +37,7 @@ const isInView = (win: Electron.BrowserWindow): boolean => {
 
 const isMatchingHost = (_url: string, _baseUrl: string): boolean => url.parse(_url).host === url.parse(_baseUrl).host;
 
-export {isInView, isMatchingHost};
+const isUUID = (string: string) =>
```

Comment:
`isUUID` should become part of https://github.com/wireapp/wire-web-packages/tree/master/packages/commons